### DISCUSSION TITLE
Shouldn't the "Update TFT display" be a button?

### DISCUSSION THREAD

**Discussion by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#328] [General]
This entity is currently a switch, and when we turn it on it will trigger the TFT update and turn itself off right away, so it behaves more like a button than a switch... so why the choice for a switch in this case?

**Comment by Blackymas** [comment]
there is no reason for it it just happened that way ;)

but I think we can change it

**Comment by edwardtfn** [comment]
I've proposed the change on #335
